sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -173,9 +174,10 @@ def test_rfe_mockclassifier():
     assert X_r.shape == iris.data.shape
 
 
-def test_rfecv():
+@pytest.mark.parametrize("csr_container", CSR_CONTAINERS)
+def test_rfecv(csr_container):
     generator = check_random_state(0)
-    iris = load_iris()
+    iris = load_iris(csr_container)
```

Comment:
I don't think this function takes any such argument. It only takes two keyword arguments.

Reply:
yeah ! it was mistakenly got there! Will update :) Thanks